core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -165,7 +165,7 @@
     /**
      * If full stack traces are enabled in the resulting image
      */
-    @ConfigItem(defaultValue = "false")
+    @ConfigItem(defaultValue = "true")
```

Comment:
Yes, this is definitely dark magic.

Reply:
This change was unintentional. I must have made a mistake when creating this, or maybe it got missed in one of the many rebases.